### PR TITLE
Improve Birthdays Function

### PR DESCRIPTION
**Pull request by AzuraAiR**
With reference to #59

### CODE REVIEW

File: src/test/data/XmlAddressBookStorageTest/invalidPersonAddressBook.xml [hunk 1]
```diff
@@ -1,11 +1,3 @@
...
-        <email isPrivate="false">hans@example.com</email>
-        <address isPrivate="false">4th street</address>
-        <birthday isPrivate="false">010194</birthday>
-    </persons>
-</addressbook>
+
a�?�^4
Orv�����1�����WJ�=GJ��WT��r�7/(���NV� e��йθ+_{>��
��O�u%��t�F�:��*��
g��t�n{�DT�
```
Comment (yeggasd) [inline src/test/data/XmlAddressBookStorageTest/invalidPersonAddressBook.xml]: Please unpush this

Comment (yeggasd) [reply src/test/data/XmlAddressBookStorageTest/invalidPersonAddressBook.xml]: and all the encrypted data file that u didnt change the contents

**Comment by AzuraAiR** [comment]
Referencing #59, #113, #112

File: src/main/java/seedu/address/commons/events/ui/BirthdayNotificationEvent.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+package seedu.address.commons.events.ui;
+
+import java.time.LocalDate;
+
+import seedu.address.commons.events.BaseEvent;
+
+/**
+ * Represents a selection change in the Person List Panel
```
Comment (jingyinno) [inline src/main/java/seedu/address/commons/events/ui/BirthdayNotificationEvent.java]: Please remember to change the comments

File: src/main/java/seedu/address/ui/InfoPanel.java [hunk 3]
```diff
@@ -54,22 +50,34 @@ public void freeResources() {
...
     }
 
     @Subscribe
     private void handleBirthdayListEvent(BirthdayListEvent event) {
         logger.info(LogsCenter.getEventHandlingLogMessage(event));
 
-        birthdayPlaceholder.getChildren().removeAll();
-        birthdayList = new BirthdayList(event.getBirthdayList());
-        birthdayPlaceholder.getChildren().add(birthdayList.getRoot());
+        //birthdayPlaceholder.getChildren().removeAll();
```
Comment (jingyinno) [inline src/main/java/seedu/address/ui/InfoPanel.java]: if these are not used, should delete them

**Pull request merged**